### ISSUE TITLE
Training of rotation network.

### ISSUE THREAD

**Issue by Hyeokreal** [naurril/SUSTechPOINTS#15]
Really appreciate for sharing the awesome repository.

I have a question on details of training rotation networks.

Could you share how have you trained the networks for rotation(yaw) prediction?
I can check the network structure within the code but hard to figure out what loss and ground-truth that you give while training.

Thank you so much !

**Comment by naurril** [comment]
The problem is defined as, given points of an object represented in a (any) local coordinate system, the network tells the heading direction in this local coordinate system. 

The train set is built from KITTI dataset.  All objects were cropped out (with size a little larger than the object),  rotated to 0 degree,  centered, and ground points removed.  During training, these objects were randomly rotated, translated (a little), and points sampled before fed into the network. The ground truth is the yaw angle after rotation, classified into 120 categories (3 degrees for each category). A classification header was attached to PointNet backbone.  So the loss is simpley categorical crossentropy loss. I also tried regression header but with worse result.

**Comment by Hyeokreal** [comment]
@naurril  Thank you so much for your comment !
